src/main/io/osd.c

```diff
@@ -454,6 +456,14 @@ void osdFormatVelocityStr(char* buff, int32_t vel, bool _3D, bool _max)
     }
 }
 
+/**
+ * Returns the average velocity. This always uses stats, so can be called as an OSD element later if wanted, to show a real time average
+ */
+void osdGenerateAverageVelocityStr(char* buff) {
```

Comment:
I think we can make it `static` and local to the osd.c. There is no need to expose that function to other modules

Reply:
I've made this change. Will test with flight data soon, hopefully today.